phpBB/styles/prosilver/template/ucp_agreement.html

```diff
@@ -33,7 +33,7 @@
 		<div class="content">
 			<h2 class="sitename-title">{SITENAME} - {L_REGISTRATION}</h2>
 			<!-- EVENT ucp_agreement_terms_before -->
-			<p><!-- IF S_SHOW_COPPA -->{L_COPPA_BIRTHDAY}<!-- ELSE -->{L_TERMS_OF_USE}<!-- ENDIF --></p>
+			<div class="agreement-text"><!-- IF S_SHOW_COPPA -->{L_COPPA_BIRTHDAY}<!-- ELSE -->{L_TERMS_OF_USE}<!-- ENDIF --></div>
```

Comment:
In this case the language key **L_COPPA_BIRTHDAY** already got a `P` tag in the php file but is wrapped also with the new CSS class `agreement-text` which BTW is not a standard P tag. Same applies for **TERMS_OF_USE_CONTENT** (AKA **L_TERMS_OF_USE**).